src/lektorium/repo/local/repo.py

```diff
@@ -204,11 +112,16 @@ def unpark_session(self, session_id):
 
     def create_site(self, site_id, name, owner=None):
         owner, email = owner or self.DEFAULT_USER
-        master_path = self.root_dir / site_id / 'master'
-        self.lektor.create_site(name, owner, master_path)
+        result = self.storage.create_site(self.lektor, name, owner, site_id)
+        site_root, site_options = result
```

Comment:
Why the need for intermediate variable `result`?

Reply:
Only readabilty. I did not find better code shape. I will remove it.